react-app/src/session-log-components/StudentSession/StudentSession.tsx

```diff
@@ -26,6 +29,17 @@ const StudentSession = ({
   const [collapsed, setCollapsed] = useState<boolean>(collapse);
   const [openSaveModal, setOpenSaveModal] = useState<boolean>(false);
   const [openCancelModal, setOpenCancelModal] = useState<boolean>(false);
+  const [user, setUser] = useState<RISEUser>();
+  const [role, setRole] = useState<"MENTOR" | "TUTOR">("MENTOR");
+  useEffect(() => {
```

Comment:
The type is not based on the current user, rather it is based on the type of the person who made the log which is log.type